editor/property_editor.cpp

```diff
@@ -818,10 +818,12 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
 				color_picker->connect("color_changed", this, "_color_changed");
 			}
 
+			float upscale = EDSCALE < 1 ? 1 : EDSCALE;
```

Comment:
It's there to keep the UV picker's minimum size at 256x256 so that users can pick from 256 different values in VeryLoDPI. If it's not desired, I'll change it to use EDSCALE directly.